src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -365,15 +365,16 @@ LedgerMaster::getFullValidatedRange (std::uint32_t& minVal, std::uint32_t& maxVa
     if (!maxVal)
         return false;
 
+    boost::optional<std::uint32_t> maybeMin;
```

Comment:
I'm going with the idea that it's not worth changing the interface here to make `minVal` an `optional`.